x-pack/test/functional/es_archives/uptime/full_heartbeat/mappings.json

```diff
@@ -1901,6 +3046,150 @@
+                },
+                "issuer": {
+                  "type": "keyword",
+                  "ignore_above": 1024
+                },
+                "ja3s": {
+                  "type": "keyword",
+                  "ignore_above": 1024
+                },
+                "not_after": {
+                  "type": "date"
+                },
+                "not_before": {
+                  "type": "date"
+                },
+                "subject": {
+                  "type": "keyword",
+                  "ignore_above": 1024
+                },
+                "x509": {
+                  "properties": {
+                    "alternative_names": {
+                      "type": "keyword",
+                      "ignore_above": 1024
+                    },
+                    "issuer": {
+                      "properties": {
+                        "common_name": {
+                          "type": "keyword",
+                          "fields": {
+                            "text": {
+                              "type": "text",
+                              "norms": false,
+                              "analyzer": "simple"
+                            }
+                          },
+                          "ignore_above": 1024
+                        },
+                        "distinguished_name": {
+                          "type": "keyword",
+                          "ignore_above": 1024
+                        }
+                      }
+                    },
+                    "not_after": {
+                      "type": "date"
+                    },
+                    "not_before": {
+                      "type": "date"
+                    },
+                    "public_key_algorithm": {
+                      "type": "keyword",
+                      "ignore_above": 1024
+                    },
+                    "public_key_curve": {
+                      "type": "keyword",
+                      "ignore_above": 1024
+                    },
+                    "public_key_exponent": {
+                      "type": "long"
+                    },
+                    "public_key_size": {
+                      "type": "long"
+                    },
+                    "serial_number": {
+                      "type": "keyword",
+                      "ignore_above": 1024
+                    },
+                    "signature_algorithm": {
+                      "type": "keyword",
+                      "ignore_above": 1024
+                    },
+                    "subject": {
+                      "properties": {
+                        "common_name": {
+                          "type": "keyword",
+                          "ignore_above": 1024
+                        },
+                        "distinguished_name": {
+                          "type": "keyword",
+                          "ignore_above": 1024
+                        },
+                        "subject": {
```

Comment:
Let's fix the double nesting of  `subject` here. I know it's a heartbeat bug, but we should fix it here separately.